nnstreamer/tensor_filter/meson.build

```diff
@@ -22,15 +22,15 @@ filter_subplugin_install_dir = subplugin_install_prefix / 'filters'
 shared_library('nnstreamer_filter_nntrainer',
   nnstreamer_filter_nntrainer_sources,
   dependencies: nnstreamer_filter_nntrainer_deps,
-  include_directories: nntrainer_inc,
+  include_directories: [nntrainer_inc, '.'],
```

Comment:
please add a comment here to not install this header file.

Reply:
Done